templates/problem/voting-stats.html

```diff
@@ -0,0 +1,146 @@
+<style>
+    .vote-stats-background{
```

Comment:
```suggestion
    .vote-stats-background {
```
Nit: space before `{`. Please update for everything else as well.

Reply:
Fixed.